docs/docs/genai/prompt-registry/adapt-prompts.mdx

```diff
@@ -0,0 +1,362 @@
+---
+sidebar_position: 6
+sidebar_label: Rewrite Prompts 🆕
+---
+
+import { APILink } from "@site/src/components/APILink";
+
+# Rewrite Prompts (Experimental)
```

Comment:
is there a reason we use `rewrite` instead of `adapt`?

Reply:
Good point, actually, I changed "adapt" to "rewrite" since I thought "Adapt prompt" is difficult to imagine the meaning. Do you think "Adapt prompt" is something many people can understand?

Reply:
I'd prefer `Adapt`. `Rewrite` sounds a bit strange because whatever we do, we end up rewriting the prompt, right?